File: UploadCodeOnPastebin.php
<?php

$url = "https://pastebin.com/api/api_post.php";

$curl = curl_init($url);
curl_setopt($curl, CURLOPT_URL, $url);
curl_setopt($curl, CURLOPT_POST, true);
curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);

$headers = array(
   "Content-Type: application/x-www-form-urlencoded",
);
curl_setopt($curl, CURLOPT_HTTPHEADER, $headers);

$data = "api_dev_key=80b366e17841431960b282260903afb6&api_paste_code=import math;print(math.pi)&api_paste_name=GitHub&api_paste_private=0&api_paste_expire_date=1Y&api_option=paste";

curl_setopt($curl, CURLOPT_POSTFIELDS, $data);

//for debug only!
curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, false);
curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);

$resp = curl_exec($curl);
curl_close($curl);
var_dump($resp);

?>

File: UploadCodeOnPastebin.py

import requests
from requests.structures import CaseInsensitiveDict

url = "https://pastebin.com/api/api_post.php"

headers = CaseInsensitiveDict()
headers["Content-Type"] = "application/x-www-form-urlencoded"

data = '''api_dev_key=XXXXXXXXXXXXXXXXXXXXXXX&api_paste_name=GitHub&api_paste_private=0&api_paste_expire_date=1Y&api_option=paste&api_paste_code=
import math
print(math.pi)
'''

resp = requests.post(url, headers=headers, data=data)

print(resp.text)
